code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/ballistic/automatic/pistol
 	name = "makarov pistol"
-	desc = "A small, easily concealable 9mm handgun. Has a threaded barrel for suppressors."
+	desc = "A small, easily concealable 9mm PMm handgun. Has a threaded barrel for suppressors."
```

Comment:
why the lowercase m? And I think technically it should either be called a PMM or a Makarov, but not both

Reply:
Small oversight from mine, looks like Modernizirovannyy is capitalized. Whoops.
Also, Makarov is the pistol's name, PM is the model.